### DISCUSSION TITLE
Welcome to Hacktober-Fest-2022 Discussions!

### DISCUSSION THREAD

**Discussion by umarwaseeem** [umarwaseeem/DSA-Hacktober-Fest-2022#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Samadmemon991** [comment]
why is this repo excluded from hacktoberfest?

**Reply by umarwaseeem** [reply]
> why is this repo excluded from hacktoberfest?

What do you mean? Can you show me screenshot?

**Reply by Samadmemon991** [reply]
![image](https://user-images.githubusercontent.com/57800880/193989941-e18be695-bf96-45cf-a67e-49dee4ed836e.png)
 Just noticed it a few moments ago

**Reply by umarwaseeem** [reply]
> ![image](https://user-images.githubusercontent.com/57800880/193989941-e18be695-bf96-45cf-a67e-49dee4ed836e.png)
> 
>  Just noticed it a few moments ago

Im sorry for that. I have been maintaining the project well but dont know why they think the project doesnt follow hacktober fest values.

Anyways thanks for any contribution you made. And thanks for letting me know.
